docs/admin/tutorial.md

```diff
@@ -124,69 +108,40 @@ then the program can be loaded with those additional return values using the `pr
 parameter (see `bpfctl help load` for list of valid values):
 
 ```console
-sudo ./target/debug/bpfctl load -p xdp -i eth0 -s "xdp" --proceed-on "drop" --proceed-on "pass" --proceed-on "dispatcher_return" --priority 150 /home/dave/dev/xdp-tutorial/basic01-xdp-pass/xdp_pass_kern.o
+sudo ./target/debug/bpfctl load --path /usr/local/src/xdp-tutorial/basic01-xdp-pass/xdp_pass_kern.o --section-name "xdp" xdp --iface vethb2795c7 --priority 150 --proceed-on "pass" --proceed-on "dispatcher_return"
 b2f19b7b-4c71-4338-873e-914bd8fa44ba
 ```
 
-Which results in (see slot 2):
+Which results in (see position 2):
```

Comment:
Maybe adding an "index" to the list call would actually be nice here? (Not in this PR but in general)